doc/HACKING/ReleasingTor.md

```diff
@@ -142,33 +146,47 @@ new Tor release:
    master, merge it with "-s ours" to avoid a needless version bump.
 
 2. Make distcheck, put the tarball up in somewhere (how about your
-   homedir on your homedir on people.torproject.org?) , and tell `#tor`
-   about it. Wait a while to see if anybody has problems building it.
-   (Though jenkins is usually pretty good about catching these things.)
+   homedir on your homedir on people.torproject.org?) , and tell `#tor-dev`
+   about it.
+
+   If you want, wait until at least one person has built it
+   successfully.  (We used to say "wait for others to test it", but our
+   CI has successfully caught this kind of errors for the last several
+   years.)
+
+
+3. Make sure that the new version is recommened in the latest consensus.
+   (Otherwise, users will get confused when it complains to them
+   about its status.)
 
 === IV. Commit, upload, announce
 
 1. Sign the tarball, then sign and push the git tag:
 
         gpg -ba <the_tarball>
-        git tag -u <keyid> tor-0.3.x.y-status
-        git push origin tag tor-0.3.x.y-status
+        git tag -s tor-0.4.x.y-status
+        git push origin tag tor-0.4.x.y-status
```

Comment:
Maybe put `status` like this `-<status>` because I actually got worried for a second that our tags had -status in them.

Reply:
d336b62fd9d811c72ec8e5929804039285616102 makes this change